examples/web/index.html

```diff
@@ -498,6 +498,20 @@ <h2>CheckboxField</h2>
           </div>
         </div>
       </section>
+
+      <section>
+        <div class="Container">
+          <h2 class="typography-heading-medium-text">Alert</h2>
+          <div class="Alert Alert--success" role="alert">
+            <div class="Alert__icon"></div>
```

Comment:
Why not a pseudo element on the whole alert? We would avoid the empty tag. Or it's like this so it is easier to add an icon when we had them ready?

@adamkudrna what do you think?

Reply:
I was thinking about this as it should be a slot for icons when it comes to the feature that icons in the alert are interchangeable. And I was not so sure about placing the icon as a background image or inline it as SVG.

Reply:
I see two ways to go:

1. Icons will be interchangeable—a temporary empty `div` is OK.
2. Icons will _not_ be interchangeable and will be always tied to `Alert` type—then we can simply set them as background image of `Alert` itself (pseudo-element is not necessary).

Reply:
Let's check what designers think about this.

Reply:
@adamkudrna @crishpeen Prefered solution from design is to have an icon interchangeable. But we had a long discussion with design and did to agree on it. Because changing icons in alerts can cause misunderstanding from users of the product.

So I suggest this approach:
- make alert with permanent icon and merge this to have alert done
- make icon interchangeable when will it be necessary, eg. do a new feature when we will need it

We both with @adamkudrna agreed that it is better to be defensive and add features when we need them.

Reply:
@adamkudrna @crishpeen So I've refactored this and removed `icon` and `message` styles.